test/integration/integration_test.go

```diff
@@ -31,21 +31,22 @@ import (
 )
 
 const (
-	datafile   = "/tmp/datafile"
-	datadir    = "/tmp/datadir"
-	configFile = "../../config.json"
+	datafile       = "/tmp/datafile"
+	datadir        = "/tmp/datadir"
+	configFile     = "../../config.json"
+	zoneConfigFile = "features/array-config/multi-az"
 )
 
 var grpcClient *grpc.ClientConn
 
-func init() {
+func readConfigFile(configurationFile string) {
 	/* load array config and give proper errors if not ok*/
-	if _, err := os.Stat(configFile); err == nil {
-		if _, err := os.ReadFile(configFile); err != nil {
-			err = fmt.Errorf("DEBUG integration pre requisites missing %s with multi array configuration file ", configFile)
+	if _, err := os.Stat(configurationFile); err == nil {
```

Comment:
This seems like an unnecessary change. It's a common idiom to use config to mean configuration.

Reply:
@donatwork Since we have introduced a new config file for the multi-az Integration tests and we want to reuse the same function (readConfigFile) for both integration tests, so we are now passing the config file as the argument, so I have used the argument name as configurationFile. 

configFile is constant that still points to the path of the config file.